server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMirroringServicePlugin.java

```diff
@@ -80,6 +80,7 @@ public DefaultMirroringService mirroringService() {
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
+                          .omitNullValues()
```

Comment:
`target()` doesn't return `null` so we don't have to add `omitNullValues`.

Reply:
Oops, it is added accidentally. Thanks for catching it.